src/lib/components/commons/FilterPanel/FilterPanel.types.ts

```diff
@@ -0,0 +1,6 @@
+
+export type IFilterPanelProps= {
+
+};
+
+export type StatusState = boolean | string;
```

Comment:
i don't like this type. it could be anything.. either make it string literal union:
'cancel' | 'active' etc...
or make it only boolean.

in any case , if we need type with those 3 options:
- 'cancel'
- true
- false
then it would be: boolean | 'cancel' and not boolean | string